src/components/Board.js

```diff
@@ -16,14 +16,56 @@ class Board extends Component {
     };
   }
 
+  componentDidMount() {
+    this.getCards()
+  }
+
+  getCards = () => {
+    axios.get('https://inspiration-board.herokuapp.com/boards/Naheed/cards')
```

Comment:
it would be nice if you used the `prop`s passed into `Board` from `App` -- `url` and `boardName`!